plugins/linguist-backend/package.json

```diff
@@ -9,6 +9,17 @@
     "main": "dist/index.cjs.js",
     "types": "dist/index.d.ts"
   },
+  "exports": {
```

Comment:
Can probably clean up these changes in the package.jsons -  they're not needed right as there's no alpha exports anymore?